src/components/MonthlyCalendar/readme.md

```diff
@@ -185,7 +190,358 @@ const initialState = {
+    font-size: 14px;
+    color: ${props => props.text.header};
+`;
+
+function DailyTasks(props) {
+    const {
+        availableTasksCount, assignedTasksCount
+    } = props;
+
+    if (!availableTasksCount && !assignedTasksCount) return null;
+
+    return (
+        <StyledContainer>
+            {assignedTasksCount > 0 &&
+                <StyledAssignedLabel>
+                    {`${assignedTasksCount} Assigned`}
+                </StyledAssignedLabel>
+            }
+            {availableTasksCount > 0 &&
+                <StyledAvailableLabel count={availableTasksCount}>
+                    {`${availableTasksCount} Availables`}
+                </StyledAvailableLabel>
+            }
+        </StyledContainer>
+    );
+}
+
+function areDatesEqual(date1, date2) {
+    const first = new Date(date1);
+    const second = new Date(date2);
+
+    return first.getYear() === second.getYear()
+        && first.getMonth() === second.getMonth()
+        && first.getDate() === second.getDate();
+}
+
+function getAvailableTasksCountForDate(date, tasks) {
+    return tasks.filter(task => areDatesEqual(date, task.date) && !task.isAssigned).length;
+}
+
+function getAssignedTasksCountForDate(date, tasks) {
+    return tasks.filter(task => areDatesEqual(date, task.date) && task.isAssigned).length;
+}
+
+function getDrawerTitle(selectedDate) {
+    const option = { month: 'long', day: 'numeric' }
+    return selectedDate ? new Intl.DateTimeFormat( 'en-US', option).format(selectedDate) : null
+}
+
+const TasksBasicInformation = ({ count, name, title }) => count ? (
+    <StyledStatisticsContainer> 
+        {name}
+        <Badge className="rainbow-m-around_medium" label={count} title={title} />
+    </StyledStatisticsContainer> ) : null;
+
+function TaskInformation({task}) {
+    return (
+            <StyledInformationContainer>
+                    <StyledHeaderContainer>
+                        <Avatar icon={<CalendarIcon />} size="medium" backgroundColor="#f4f6f9" />
+                        <StyledHourContainer>{task.hour}</StyledHourContainer>
+                    </StyledHeaderContainer>
+                    <StyledContentContainer>
+                        {task.description}
+                    </StyledContentContainer>
+                    {task.isConfirmed ?
+                        <StyledContentContainer>
+                            <StyledIconContainer>
+                                <Avatar icon={<CheckmarkIcon />} size="x-small" backgroundColor="#1de9b6" />
+                            </StyledIconContainer>
+                            <StyledStatusText>Confirmed</StyledStatusText>
+                        </StyledContentContainer> :
+                        <StyledContentContainer>
+                            <StyledStatusText> not confirmed</StyledStatusText>
+                        </StyledContentContainer>
+                    }
+            </StyledInformationContainer>
+    );
+}
+
+function GetDrawerTasks({date, tasks}) {
```

Comment:
I think you should change the name to DrawerTasks and it should be a component like this

`const DrawerTasks = props => {
    const { date, tasks } = props;

    const assignedTasks = tasks.filter(task => areDatesEqual(date, task.date) && task.isAssigned);
    const availableTasks = tasks.filter(task => areDatesEqual(date, task.date) && !task.isAssigned);

    return (
        <>
            <TasksBasicInformation
                count={assignedTasks.lenght}
                name="ASSIGNED"
                title="AssignedTasks"
            />
            {assignedTasks.map(task => (
                <TaskInformation {...task} />
            ))}
            <TasksBasicInformation
                count={availableTasks.lenght}
                name="AVAILABLE"
                title="AvailableTasks"
            />
            {availableTasks.map(task => (
                <TaskInformation {...task} />
            ))}
        </>
    );
};`